Duplicating Bean on Spring Boot Example for AppInsights

fvztdk opened the pull request:
You should not use new on this case, just calling the method will use the already created instance of the Bean

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

4c74356b41 commented on articles/site-recovery/site-recovery-active-directory.md:
```diff
@@ -78,19 +78,19 @@ Then [reconfigure the DNS server for the virtual network](../active-directory/ac
 ## Test failover considerations
 
-Test failover occurs in a network that's isolated from production network so that there's no impact to production workloads.
+Test failover occurs in a network that's isolated from production network so that there's no impact on production workloads.
```
I think it's more correct this way

PRmerger commented:
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @prateek9us, has been notified to review your proposed change.

prateek9us commented:
@4c74356b41 Thanks for the contribution. 
@PRmerger  This can be merged as-is.

prateek9us commented:
#sign-off

4c74356b41 commented:
thanks

fvztdk committed:
Duplicating Bean

You should not use new on this case, just calling the method will use the already created instance of the Bean
